### PR TITLE
Document GUI Versioning policy

### LINKED ISSUE #327: Document how TestCentric GUI is versioned
Could be part of an existing doc or a new one.

### PR DESCRIPTION
**Pull request by CharliePoole**
Fixes #327

### CODE REVIEW

**Comment by CharliePoole** [comment]
Since it's a new file, I'll go ahead and merge so as to close the issue for beta2, but I'll still read comments for an eventual update.

File: README.md [hunk 1]
```diff
@@ -21,8 +21,12 @@ The **chocolatey** package provides the best user experience and is the recommen
 
 To use the **zip** distribution, you should simply unzip the contents into a convenient directory and create your own shortcut to `testcentric.exe`. To use it from the command-line, place the install directory on your path.
 
+## Versioning
+
+**TestCentric** uses semantic versioning for it's version numbers, with specific exceptions and clarification regarding __what is considered a breaking change__. See the [VERSIONING](./VERSIONING.md) file for more information.
```
Comment (mikkelbu) [inline README.md]: * it's => its
* also a bit inconsistent both to use ** and __ for marking text as bold

File: VERSIONING.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+1. The GUI interface itself, as presented to the user.
+2. The command-line interface to the GUI.
+
+### User Interface
+
+The GUI presents the user with information (output) in a graphical format and allows the user to specify actions like running tests by interacting with it. In general, we are concerned about preserving the capabilities, which the GUI provides to the user, much more than we are about the specific appearance of the GUI elements. However, those elements may also be important in certain cases.
+
+#### Examples of MAJOR UI Changes
+
+- Removal of a capability. For example, if we were to remove the abiity, now provided, to automatically reload tests before running.
```
Comment (mikkelbu) [inline VERSIONING.md]: abiity => ability

File: VERSIONING.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+- Changing an existing UI element so it does something completely different.
+
+#### Examples of MINOR UI Changes
+
+- Provision of a new capability. F1or example, adding the ability to save test results in multiple formats.
```
Comment (mikkelbu) [inline VERSIONING.md]: F1or => For

File: README.md (hunk 1 above)
Comment (CharliePoole) [reply README.md]: Took out the bolding... too much there I think.

**Comment by CharliePoole** [comment]
Thanks @mikkelbu - every time I switch to VsCode I end up with typos. I'll just push the fixes directly.

**Comment by CharliePoole** [comment]
Ah... seems the spellchecker was updated and I needed to reload it. If it knows enough to tell me that, you think it could do it!

**Pull request merged**